docs/resources/target_group.md

```diff
@@ -59,6 +59,7 @@ The following arguments are supported:
     - `ip` - (Required)[string] The IP of the balanced target VM.
     - `port` - (Required)[int] The port of the balanced target service; valid range is 1 to 65535.
     - `weight` - (Required)[int] Traffic is distributed in proportion to target weight, relative to the combined weight of all targets. A target with higher weight receives a greater share of traffic. Valid range is 0 to 256 and default is 1; targets with weight of 0 do not participate in load balancing but still accept persistent connections. It is best use values in the middle of the range to leave room for later adjustments.
+    - `proxy_protocol` - (Optional)[string] The proxy protocol version. Accepted values are `none`, `v1`, `v2`, `v2ssl`. If unspecified, the default value of `none` is used.
```

Comment:
@cbeti-ionos I suggest also adding this in the example as you did for `docs/resources/networkloadbalancer_forwardingrule.md`

Reply:
done